dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -205,34 +208,10 @@ export const JobDetailChartsPage = () => {
         >
           <NodeCountCard className={classes.nodeCountCard} />
           <Section flex="1 1 500px">
-            <Box
-              style={{
-                overflow: "hidden",
-                overflowY: "scroll",
-              }}
-              sx={{ borderRadius: "16px" }}
-              marginLeft={1}
-              marginRight={1}
-            >
-              {cluster_status?.data
-                ? formatNodeStatus(cluster_status?.data.clusterStatus)
```

Comment:
since we moved `formatNodeStatus` to the other file. Can we delete the function from this file?

Reply:
Ohh! Good catch, I did not notice this since there is no eslint error pop up